src/utils.js

```diff
@@ -1 +1,6 @@
-export const isEmptyObject = obj => obj ? Object.keys(obj).length === 0 && obj.constructor === Object : true;
\ No newline at end of file
+export const isEmptyObject = obj => obj ? Object.keys(obj).length === 0 && obj.constructor === Object : true;
+export const getRandomPariority = priorities => {
+    if ((typeof priorities) !== 'object' || priorities.constructor !== Array) return;
+    const ri = Math.floor(Math.random() * priorities.length)
+    return priorities[ri]
```

Comment:
please dont forget to put `;`

Reply:
Also where is this function being used?

Reply:
That was a utility function that I created but later went ahead to implement it in the `visulaizer.js` file directly to maintain the context of use. This function is now refactored and removed.